yarn.lock

```diff
@@ -899,9 +899,9 @@ agentkeepalive@^4.1.2:
     depd "^1.1.2"
     humanize-ms "^1.2.1"
 
-"agentkeepalive@git+https://github.com/pagopa/agentkeepalive.git#v4.1.1":
+"agentkeepalive@https://github.com/pagopa/agentkeepalive#v4.1.1":
```

Comment:
this should not be here, can you check using `yarn why agentkeepalive` where does this dep come from? we must use the official repo now, not the forked clone, probably you need to upgrade some package (ie. io-functions-commons or io-ts-commons). check that all packages are updated to the latest version